Core/src/org/sleuthkit/autopsy/directorytree/DirectoryTreeFilterChildren.java

```diff
@@ -120,7 +121,7 @@ private static boolean isLeafDirectory(DirectoryNode node) {
                         && !((Directory) c).getName().equals(".."))) {
                     ret = false;
                     break;
-                } else if (c.hasChildren()) {
+                } else if (0 < Case.getCurrentCase().getSleuthkitCase().getVisibleContentChildrenCount(c)) {
```

Comment:
Not easily I don't think - I also wanted to make that change, but hasVisibleChildren is on AbstractContentNode not Content